concepts/05 Widgets/Scheduler/00 Getting Started with Scheduler/10 Configure Views.md

```diff
@@ -0,0 +1,149 @@
+The **Scheduler** supports the following views:
+
+* [Day](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Day_View) (default)
+
+* [Week and WorkWeek](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Week_and_WorkWeek_Views) (week is added by default)
+
+* [Month](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Month_View)
+
+* [Timeline](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Timeline_Views)
+
+* [Agenda](/Documentation/Guide/Widgets/Scheduler/Views/View_Types/#Agenda_View)
+
+* [Customized views](/Documentation/Guide/Widgets/Scheduler/Views/Customize_Individual_Views/).
+
+The default views are *"day"* and *"week"*. To use custom ones, specify the [views[]](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) property. In this tutorial, we will use *"day"*, *"week"*, *"month"*, and *"timelineWeek"* views.
```

Comment:
Never use "one" as a substitution for a noun in docs. If you feel the urge to do this, rephrase the sentence.

> The default views are *"day"* and *"week"*.

and then

> The default view is *"day"*.

How it this possible?